src/main/scala/uk/gov/hmrc/play/http/HeaderCarrier.scala

```diff
@@ -33,12 +34,14 @@ case class HeaderCarrier(authorization: Option[Authorization] = None,
                          requestId: Option[RequestId] = None,
                          requestChain: RequestChain = RequestChain.init,
                          nsStamp: Long = System.nanoTime(),
-                         extraHeaders: Seq[(String, String)] = Seq(), 
+                         extraHeaders: Seq[(String, String)] = Seq(),
```

Comment:
"extraHeaders" have been explicitly added by the calling client (i.e. http-verbs user). "otherHeaders" are the "all the rest that are not blacklisted" which are now being passed-through.